### DISCUSSION TITLE
Difference between brew install --cask [cask-name] and brew install [cask-name]

### DISCUSSION THREAD

**Discussion by vizzale** [Homebrew/discussions#480] [Casks] [answered]
Hi, I was wondering if there is any difference in installing a Cask, like Google Chrome, like this:
`brew install --cask google-chrome` (as suggested here https://formulae.brew.sh/cask/google-chrome#default)
or in this way:
`brew install google-chrome` (as suggested here https://github.com/Homebrew/homebrew-cask/blob/master/USAGE.md)

What's the difference? Or is the result identical?
Thank you in advance for your reply.

**Comment by SMillerDev** [comment, marked as answer]
The result is identical here, but for appium (https://formulae.brew.sh/formula/appium | https://formulae.brew.sh/cask/appium) only the former would install the cask version.

**Reply by vizzale** [reply]
So if I understand, the need to make the argument explicit `--cask` occurs only when there is a "formula" and a "cask" version of the application. In the case of applications such as "Google Chrome", "Firefox", etc.. the two versions of the command are equivalent. It's correct?

**Reply by SMillerDev** [reply]
Yes, `--cask` can be used as a disambiguation.

**Reply by vizzale** [reply]
@SMillerDev Thank you very much for this explanation.
